src/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit/Models/Kepler.cpp

```diff
@@ -742,16 +745,16 @@ State                           Kepler::CalculateJ4StateAt                  (
         1.0 + expr * (2.0 + eccentricityAtEpochSquared / 2.0 - 2.0 * sqrtBeta 
```

Comment:
@robinpdm just checking but the Beta here is not meant to be the Beta variable defined in Vallado correct ?

Reply:
Here, `sqrtBeta = std::sqrt(1.0 - eccentricityAtEpoch * eccentricityAtEpoch)`